### PR TITLE
Removes ERT Rangers items from loadout

### LINKED ISSUE #8375: ERT-Ranger Costume is in the loadout (And on stowaways)
#### Brief description of the issue
Parts of the ERT Ranger costume are spread all around the loadout.
#### What you expected to happen
The ERT-Ranger Costumes, which IC where handmade, and OOC are kind of buggy messes of missing sprites.
#### What actually happened
Buggy versions of the white one found in the loadout selection, as 'Jumpsuit, Selection', 'Hardhat, Selection', possible more if other 'item, Selections' get added, for example shoes or bags or so....
#### Steps to reproduce
Just check the loadout. Maybe try spawning with them (have a accessory handy so you can actually unequip.)
#### Additional info:
 - **Anything else you may wish to add** I have been told that it is because how selections work, and wonder if that selections do not have a option to exclude chosen items is a issue in itself?

### PR DESCRIPTION
**Pull request by Atermonera**
Fixes #8375

### CODE REVIEW

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -1275,34 +1275,34 @@ Uniforms and such
 //Ranger uniforms
 //On-mob sprites go in icons\mob\uniform.dmi with the format "white_ranger_uniform_s" - with 'white' replaced with green, cyan, etc... of course! Note the _s - this is not optional.
 //Item sprites go in icons\obj\clothing\ranger.dmi with the format "white_ranger_uniform"
-/obj/item/clothing/under/color/ranger
+/obj/item/clothing/under/ranger
```
Comment (Spookerton) [inline code/modules/clothing/under/miscellaneous.dm:1278]: under/color sets
```
rolled_sleeves = 0
```

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 2]
```diff
@@ -174,34 +174,34 @@
 	drop_sound = 'sound/items/drop/clothing.ogg'
 	pickup_sound = 'sound/items/pickup/clothing.ogg'
 
-/obj/item/clothing/shoes/boots/ranger
+/obj/item/clothing/shoes/ranger
```
Comment (Spookerton) [inline code/modules/clothing/shoes/miscellaneous.dm:177]: shoes/boots sets
```
force = 3
can_hold_knife = 1
step_volume_mod = 1.2
drop_sound = 'sound/items/drop/boots.ogg'
```

File: code/modules/clothing/head/misc.dm [hunk 3]
```diff
@@ -502,4 +502,40 @@
 	desc = "A marine helmet prop from the popular game 'Ruin'."
 	icon_state = "marine"
 	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|BLOCKHAIR
-	body_parts_covered = HEAD|FACE|EYES
\ No newline at end of file
+	body_parts_covered = HEAD|FACE|EYES
+
+/obj/item/clothing/head/ranger
```
Comment (Spookerton) [inline code/modules/clothing/head/misc.dm:507]: head/hardhat also sets
```
brightness_on = 4
armor = list(
	melee = 30,
	bullet = 5,
	laser = 20,
	energy = 10,
	bomb = 20,
	bio = 10,
	rad = 20
)
action_button_name = "Toggle Head-light"
w_class = ITEMSIZE_NORMAL
ear_protection = 1
```

**Review by Spookerton — CHANGES_REQUESTED** [review]
probably unintended mechanics changes

File: code/modules/clothing/head/misc.dm (hunk 3 above)
Comment (Atermonera) [reply code/modules/clothing/head/misc.dm:507]: Actually mostly deliberate, this is a costume item, so does it really need _any_ gizmos/armor?

File: code/modules/clothing/shoes/miscellaneous.dm (hunk 2 above)
Comment (Atermonera) [reply code/modules/clothing/shoes/miscellaneous.dm:177]: Also somewhat intended, nerfing a costume item.

**Review by Spookerton — APPROVED** [review]
works for me!

**Pull request merged**